### ISSUE TITLE
dcm2mnc: update manpage to document %P for using protocol name in mnc file naming

### ISSUE THREAD

**Issue by gdevenyi** [BIC-MNI/minc-tools#61]
Something very nice about modern dcm2nii converters is the output files can be named with the sequence information from the DICOMs.

Right now dcm2mnc only offers the acquisition number as a naming option. Sadly, if the sequence was done out-of-order at the scanner this number is unreliable.

Currently I rely on parsing the mincheader output to rename the resulting files to something more useful, since I commonly encounter scanning sessions with many sequences.

**Comment by andrewjanke** [comment]
This is already possible in dcm2mnc, have a look at the -fname and -dname options. Here's python snippet showing what is possible.

cmd = ['dcm2mnc', '-usecoordinates', '-anon',
                '-dname', '',
                '-fname', '%N_%D-%T_' + str(SeriesNumber).zfill(2) + '_' + StudyDescription + '_' + SeriesDescription + '%s%e%t%p%c']
            cmd.extend(flist)
            cmd.append(outdir)
            do_cmd(cmd, False)

(From here: https://github.com/NIF-au/imagetrove-python-uploader/blob/master/dicom/imgtr-uploader-dicom.py#L421)

a

**Comment by gdevenyi** [comment]
Thanks @andrewjanke I was aware of -fname and -dname, but it didn't appear to me that any of the "%" formatting options actually contain the info I want inserted? It looks like you're getting that elsewhere in that snippet?

**Comment by andrewjanke** [comment]
Correct,

I extract exactly what I want with dcmdump as it is dependent on the
scanner.

A

**Comment by rdvincent** [comment]
@gdevenyi There is a poorly-documented option, %P, that is available to insert the protocol name, as well as %A to insert the acquisition or series ID. Would either of these help?

**Comment by gdevenyi** [comment]
%P is exactly what I'm looking for

Recasting request as a "update manpage"

**Comment by rdvincent** [comment]
@gdevenyi It is already in the man page, at least on the develop branch. Not sure the man page is getting installed however.

**Comment by gdevenyi** [comment]
Ah, on develop I'm seeing a manpage without this option listed.

**Comment by rdvincent** [comment]
It should be listed as shown here:

https://github.com/BIC-MNI/minc-tools/blob/develop/conversion/dcm2mnc/dcm2mnc.man1#L214

**Comment by gdevenyi** [comment]
Yup, my sources have that, but the installed version seem to be coming from somewhere else.

**Comment by vfonov** [comment]
http://bic-mni.github.io/man-pages/man/dcm2mnc

**Comment by gdevenyi** [comment]
Thanks @vfonov but that man page also is missing "%P"

**Comment by vfonov** [comment]
I guess this commit came after I split up man pages: https://github.com/BIC-MNI/minc-tools/commit/157d0532dd137e21147ee3de6a92e77f37463457
